apps/opik-backend/src/main/java/com/comet/opik/infrastructure/http/CORS.java

```diff
@@ -0,0 +1,56 @@
+package com.comet.opik.infrastructure.http;
+
+import com.comet.opik.infrastructure.OpikConfiguration;
+import io.dropwizard.core.setup.Environment;
+import jakarta.servlet.DispatcherType;
+import jakarta.servlet.FilterRegistration;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.jetty.servlets.CrossOriginFilter;
+
+import java.util.EnumSet;
+
+@Slf4j
+public class CORS {
```

Comment:
Minor: let's follow the Java conventions for naming classes: `CORS` vs `Cors`. Anyway, this seems to be `CorsFactory`.

Reply:
Done in https://github.com/comet-ml/opik/pull/498/commits/8cfea8ab110709aeafa6c79f6f5e331496fe1176